src/rbd.cc

```diff
@@ -136,7 +136,7 @@ void usage()
 "  status <image-name>                         show the status of this image\n"
 "  map <image-name>                            map image to a block device\n"
 "                                              using the kernel\n"
-"  unmap <device>                              unmap a rbd device that was\n"
+"  unmap <device> | <image-name>               unmap a rbd device that was\n"
```

Comment:
for consistency with other commands this should take into account pool and snapshot too, with the existing parsing for those (i.e. it should work with pool/image@snap or --pool p --image i --snap s or combinations of those)